simulator-core/build.gradle

```diff
@@ -34,6 +34,8 @@ lombok {
 }
 
 dependencies {
+    compile project(":payload:ocpp-data-types")
```

Comment:
As far as I remember fat jar doesn't include this dependency, we published it separately as `io.everon:ocpp-payload:2.0.0`. So now if you'll update jar in repository and increase version in everon - it will be failing